src/targets/google-analytics/event-helpers.js

```diff
@@ -0,0 +1,240 @@
+export const trackPageView = (eventDefinition, tracker) => (
+  action,
+  prevState,
+  nextState
+) => {
+  const event = eventDefinition(action, prevState, nextState);
+  const { page, title, location } = event;
+
+  return {
+    hitType: 'pageview',
+    customTrackerId: tracker,
+    page,
+    title,
+    location,
+  };
+};
+
+export const trackEvent = (eventDefinition, tracker) => (
+  action,
+  prevState,
+  nextState
+) => {
+  const event = eventDefinition(action, prevState, nextState);
+  const { eventCategory, eventAction, eventLabel, eventValue } = event;
```

Comment:
one thing that caught my eye. Do you think we can change this to:

```js
const { category, action, label, value } = event;
```

Having the event in there is a bit redundant. Also, if you can update the documentation to reflect this at `docs/targets/google-analytics.md` that would be awesome.